dumpling/util.go

```diff
@@ -58,8 +58,22 @@ func parseExtraArgs(logger *log.Logger, dumpCfg *export.Config, args []string) e
 		fileSizeStr     string
 		tablesList      []string
 		filters         []string
+		newargs         []string
 	)
 
+	// compatibility for `--no-locks`
+	for i, arg := range args {
+		if arg == "--no-locks" {
+			logger.Warn("`--no-locks` is replaced by `--consistency none` since v2.0.0")
+			newargs = args[:i]
+			newargs = append(newargs, "--consistency", "none")
```

Comment:
if also specify another `--consistency`, can report an error in `dumplingFlagSet.Parse(args)`?